docs/en/install.md

```diff
@@ -14,14 +14,14 @@
 
 MMOCR has different version requirements on MMCV and MMDetection at each release to guarantee the implementation correctness. Please refer to the table below and ensure the package versions fit the requirement.
 
-| MMOCR | MMCV | MMDetection |
-| - | - | - |
-| master | 1.3.8 <= mmcv <= 1.5.0 | 2.14.0 <= mmdet <= 3.0.0 |
+| MMOCR        | MMCV                   | MMDetection               |
+| ------------ | ---------------------- | ------------------------- |
```

Comment:
Strange here. My Vscode will automatically format markdown. Hope this is not a problem

Reply:
I'm jealous of this feature